compose/service.py

```diff
@@ -1069,6 +1071,28 @@ def parse_restart_spec(restart_config):
 
     return {'Name': name, 'MaximumRetryCount': int(max_retry_count)}
 
+# Ulimits
+
+
+def build_ulimits(ulimit_config):
+    if not ulimit_config:
+        return None
+    ulimits = []
+    for limit_name, soft_hard_values in six.iteritems(ulimit_config):
+        if isinstance(soft_hard_values, six.integer_types):
+            ulimits.append({'Name': limit_name, 'soft': soft_hard_values, 'hard': soft_hard_values})
```

Comment:
Minor: it looks like `docker-py` supports both `name` as well (and it's the first option it checks). Could we use `name` (lowercase) here to be consistent with the other fields?